2025-02-02 09:16:25,085 INFO    Thread-8  :6237 [wandb_setup.py:_flush():68] Current SDK version is 0.19.4
2025-02-02 09:16:25,085 INFO    Thread-8  :6237 [wandb_setup.py:_flush():68] Configure stats pid to 6237
2025-02-02 09:16:25,085 INFO    Thread-8  :6237 [wandb_setup.py:_flush():68] Loading settings from /home/aiffel_guacamole/.config/wandb/settings
2025-02-02 09:16:25,085 INFO    Thread-8  :6237 [wandb_setup.py:_flush():68] Loading settings from /home/aiffel_guacamole/MoE/MoE/wandb/settings
2025-02-02 09:16:25,085 INFO    Thread-8  :6237 [wandb_setup.py:_flush():68] Loading settings from environment variables
2025-02-02 09:16:25,085 INFO    Thread-8  :6237 [wandb_init.py:setup_run_log_directory():624] Logging user logs to /home/aiffel_guacamole/MoE/MoE/wandb/run-20250202_091625-c782vv85/logs/debug.log
2025-02-02 09:16:25,085 INFO    Thread-8  :6237 [wandb_init.py:setup_run_log_directory():625] Logging internal logs to /home/aiffel_guacamole/MoE/MoE/wandb/run-20250202_091625-c782vv85/logs/debug-internal.log
2025-02-02 09:16:25,086 INFO    Thread-8  :6237 [wandb_init.py:init():743] calling init triggers
2025-02-02 09:16:25,086 INFO    Thread-8  :6237 [wandb_init.py:init():748] wandb.init called with sweep_config: {'batch_size': 32, 'epochs': 50, 'learning_rate': 0.0005111783086175036, 'optimizer': 'sgd', 'weight_decay': 0.0009254734120901636}
config: {}
2025-02-02 09:16:25,086 INFO    Thread-8  :6237 [wandb_init.py:init():776] starting backend
2025-02-02 09:16:25,323 INFO    Thread-8  :6237 [wandb_init.py:init():780] sending inform_init request
2025-02-02 09:16:25,325 INFO    Thread-8  :6237 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-02-02 09:16:25,330 INFO    Thread-8  :6237 [wandb_init.py:init():795] backend started and connected
2025-02-02 09:16:25,332 INFO    Thread-8  :6237 [wandb_run.py:_config_callback():1270] config_cb None None {'batch_size': 32, 'epochs': 50, 'learning_rate': 0.0005111783086175036, 'optimizer': 'sgd', 'weight_decay': 0.0009254734120901636}
2025-02-02 09:16:25,342 INFO    Thread-8  :6237 [wandb_init.py:init():888] updated telemetry
2025-02-02 09:16:25,406 INFO    Thread-8  :6237 [wandb_init.py:init():915] communicating run to backend with 90.0 second timeout
2025-02-02 09:16:25,518 INFO    Thread-8  :6237 [wandb_init.py:init():967] starting run threads in backend
2025-02-02 09:16:25,820 WARNING MsgRouterThr:6237 [router.py:message_loop():75] message_loop has been closed
2025-02-02 09:16:25,898 INFO    Thread-8  :6237 [wandb_run.py:_console_start():2409] atexit reg
2025-02-02 09:16:25,898 INFO    Thread-8  :6237 [wandb_run.py:_redirect():2259] redirect: wrap_raw
2025-02-02 09:16:25,898 INFO    Thread-8  :6237 [wandb_run.py:_redirect():2324] Wrapping output streams.
2025-02-02 09:16:25,898 INFO    Thread-8  :6237 [wandb_run.py:_redirect():2349] Redirects installed.
2025-02-02 09:16:25,899 ERROR   Thread-8  :6237 [wandb_init.py:init():1454] error in wandb.init()
Traceback (most recent call last):
  File "/home/aiffel_guacamole/anaconda3/envs/moe/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 1444, in init
    return wi.init(run_settings, run_config)
  File "/home/aiffel_guacamole/anaconda3/envs/moe/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 1008, in init
    run._on_start()
  File "/home/aiffel_guacamole/anaconda3/envs/moe/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 2453, in _on_start
    self._on_ready()
  File "/home/aiffel_guacamole/anaconda3/envs/moe/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 2496, in _on_ready
    self._telemetry_flush()
  File "/home/aiffel_guacamole/anaconda3/envs/moe/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 771, in _telemetry_flush
    self._backend.interface._publish_telemetry(self._telemetry_obj)
  File "/home/aiffel_guacamole/anaconda3/envs/moe/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 101, in _publish_telemetry
    self._publish(rec)
  File "/home/aiffel_guacamole/anaconda3/envs/moe/lib/python3.9/site-packages/wandb/sdk/interface/interface_sock.py", line 47, in _publish
    self._sock_client.send_record_publish(record)
  File "/home/aiffel_guacamole/anaconda3/envs/moe/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 222, in send_record_publish
    self.send_server_request(server_req)
  File "/home/aiffel_guacamole/anaconda3/envs/moe/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 154, in send_server_request
    self._send_message(msg)
  File "/home/aiffel_guacamole/anaconda3/envs/moe/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 151, in _send_message
    self._sendall_with_error_handle(header + data)
  File "/home/aiffel_guacamole/anaconda3/envs/moe/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 130, in _sendall_with_error_handle
    sent = self._sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe
2025-02-02 09:16:25,925 INFO    Thread-8  :6237 [wandb_run.py:_finish():2134] finishing run aiffelthon-GUACAMole/moe-hyperparameter-tuning/c782vv85
